assets/scripts/script.js

```diff
@@ -109,7 +109,36 @@
       }
 
       // Adjust styling before slider init.
-      $thumbnailSliderEl.addClass('flickity');
+      $thumbnailSliderEl
+        .addClass('flickity')
+        .on('ready.flickity', function(event) {
+          // Video should be the second element in the product gallery.
+          const $galleryWraper = $('.sidebar-single-product .woocommerce-product-gallery__wrapper');
+          const $galleryElements = $galleryWraper.children('div');
+          if ($galleryElements.length <= 1) {
+            return;
+          }
+          const $firstGalleryElement = $galleryElements.first();
+          if ($firstGalleryElement.hasClass('has-video')) {
+            $firstGalleryElement.detach().insertAfter($galleryWraper.children('div').first());
+          }
+        })
+        .on('settle.flickity', function(event, index) {
+          // Insert the video thumbnail in the thumbs slider.
+          const $videoContent = $('.sidebar-single-product .gallerya__video-content');
+          if ($videoContent.length) {
+            const $sliderThumbs = $('.sidebar-single-product .flickity-slider li img');
+            // Insert the video thumb in second position, if posible.
+            const videoThumbPos = $sliderThumbs.length > 1 ? 1 : 0;
+            const videoThumbSrc = $videoContent.data('video-thumb');
+
+            $sliderThumb = $sliderThumbs.eq(videoThumbPos);
+            $sliderThumb.attr('src', videoThumbSrc);
+            $sliderThumb.attr('lazy-src', videoThumbSrc);
+            $sliderThumb.removeClass('lazy-hidden');
```

Comment:
@juanlopez4691 what's happening here in relation to lazy load? Do we maybe need a comment to clarify these lines?